controllers/Redirects.php

```diff
@@ -138,6 +138,30 @@ public function update($recordId = null, $context = null)
         parent::update($recordId, $context);
     }
 
+    /**
+     * Force rule flushing on update
+     */
+    public function update_onSave(?string $context = null)
+    {
+        /** @noinspection PhpPossiblePolymorphicInvocationInspection */
+        parent::update_onSave($context);
+
+        $fromUrl = post('Redirect')['from_url'] ?? null;
```

Comment:
```suggestion
        $fromUrl = $this->formGetWidget()->getSaveData()['from_url'] ?? null;
```

Reply:
Added in 21326ac38c8e02d76d9e1a8845d8e52351853623